pkg/virtctl/expose/expose.go

```diff
@@ -76,8 +75,11 @@ virtualmachineinstance (vmi), virtualmachine (vm), virtualmachineinstancereplica
 }
 
 func usage() string {
-	usage := `  # Expose SSH to a virtual machine instance called 'myvm' as a port (5555) and specify each node open up port 30001 on the cluster:
-  virtctl expose vmi myvm --port=5555 --node-port=30001 --target-port=22 --name=myvm-ssh --type=NodePort")`
+	usage := `  # Expose SSH to a virtual machine instance called 'myvm' on each node via a NodePort service:
+  virtctl expose vmi myvm --port=22 --name=myvm-ssh --type=NodePort
```

Comment:
would be good to keep the port vs. target-port distinction in the example, so it is clear they can be different

Reply:
Let me check again, but I think that in the case of `NodePort`, kubectl only uses `--port` and ignores `--target-port`.